### PR TITLE
[caffe2] Handle 'axis' parameter for softmax op in caffe-caffe2 translator.

### PR DESCRIPTION
**Pull request by JerryShih**
The axis information doesn't pass to caffe2 op.

### CODE REVIEW

**Comment by JerryShih** [comment]
@bddppq 
Could you please review this change?

**Comment by JerryShih** [comment]
ping @bddppq

**Comment by JerryShih** [comment]
@bddppq @houseroad 
All tests passed.
Could you please review this patch?

**Comment by ezyang** [comment]
I can merge this one, if we can get a test for it?

**Comment by houseroad** [comment]
The test should go to https://github.com/pytorch/pytorch/blob/master/caffe2/python/caffe_translator_test.py
